packages/mask/src/plugins/Avatar/utils/index.ts

```diff
@@ -34,7 +34,7 @@ export function toPNG(image: string) {
         const ctx = canvas.getContext('2d')
         if (isNull(ctx)) throw new Error('Canvas was not supported')
         img.addEventListener('load', () => {
-            ;[canvas.width, canvas.height] = [400, 400]
+            ;[canvas.width, canvas.height] = [img.width, img.height]
```

Comment:
Why not constrain it with the maximum size.

Reply:
Rectangular image will be deformed